masonry/src/render_root.rs

```diff
@@ -74,6 +74,7 @@ pub(crate) struct RenderRootState {
     pub(crate) mutate_callbacks: Vec<MutateCallback>,
     pub(crate) is_ime_active: bool,
     pub(crate) scenes: HashMap<WidgetId, Scene>,
+    pub(crate) pointer_changed: bool,
```

Comment:
Hmm, I'm not sure if that name is slightly misleading. `needs_pointer` doesn't sound too great though, and `needs_pointer_pass` is also not 100% optimal as it is not consistent to the other `needs_` flags, but it's probably better in describing what this is for.

Reply:
Maybe `needs_pointer_update`?